source-academy/cadet-frontend
# adding support for file persistence using GitHub

### Description

Added Open, Save, Quick save features for GitHub.

### Type of change


### How to test

Deploy the front-end. Check that the GitHub control buttons now have options for "Open", "Save", "Save As" and "Log In".
Click on log in and sign in to GitHub.
Click on Open, select repository, then select a file to load.
Click on Save As, select repository, then select a file or name a new file to save over.
Click on Save to quick save current work into previously selected file.
